packages/fetchye/src/mapHeaderNamesToLowerCase.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
```

Comment:
```suggestion
 * Copyright 2021 American Express Travel Related Services Company, Inc.
```

Reply:
Changed this within the existing commit. Hope that's fine.